plugins/kubernetes-backend/api-report.md

```diff
@@ -340,7 +344,7 @@ export type KubernetesObjectTypes =
 export class KubernetesProxy {
   constructor(logger: Logger, clusterSupplier: KubernetesClustersSupplier);
   // (undocumented)
-  createRequestHandler(): RequestHandler;
+  createRequestHandler(permissionApi: PermissionEvaluator): RequestHandler;
```

Comment:
Please make this an options object, see `#9` in https://github.com/backstage/backstage/blob/master/STYLE.md#api-design

It's with the expectation that we may need additional arguments here in the future

Reply:
Alrighty refactoring to use an options object.

Reply:
Added